File: TP1 - Dart/domain/entities/person.dart
class Person {
  final String _id;
  String _name;

  Person(this._id, [this._name = '']);

  // Getter's
  String get id => this._id;
  String get name => this._name;

  // Setter's
  set name(String name) => this._name = name;
}

File: TP1 - C++/data/models/person.cpp
#include "../../domain/entities/person.hpp"

Person::Person(string id, string name) :
    id(id),
    name(name)
{}

Person::Person(string id){
    this->id = id;
    name = ' ';
}

Person::Person(){
    id = ' ';
    name = ' ';
}

Person::~Person(){}

string Person::getId(){
    return id;
}

string Person::getName(){
    return name;
}

void Person::setName(string name){
    this->name = name;
}
